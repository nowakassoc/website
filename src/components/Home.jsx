import React from 'react';
import Icon from './Icon';
import profilePic from '../assets/images/Martin_Nowak.png';
import profilePicMobile from '../assets/images/Martin_Nowak_sm.png';

const Home = () => (
  <section id="section-home">
    <div className="home-col-left">
      <span id="home" className="section-link" />
      <h1>Nowak and Associates, Inc.</h1>
      <h2 className="tagline">Healthcare and Business Consulting</h2>
      <p>
        <img
          src={profilePicMobile}
          alt="Martin C. Nowak - Nowak and Associates"
          className="home-portrait-mobile"
        />
        Nowak and Associates offers boutique health strategies for MDs and
        hospitals. Our services include personalized advice for best doctors and
        hospitals, legal work for medical malpractice and medical class action
        lawsuits, business plan development and inventor guidance.
        <br />
        <a
          aria-label="Contact Nowak and Associates"
          href="#contact"
          id="home-contact"
        >
          <Icon icon="envelope" className="icon-link" />
          Contact Nowak &amp; Associates
        </a>
      </p>
    </div>
    <div className="home-col-right">
      <img
        src={profilePic}
        alt="Martin C. Nowak - Nowak and Associates"
        className="home-portrait"
      />
    </div>
  </section>
);

export default Home;
