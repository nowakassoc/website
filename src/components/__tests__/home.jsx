import React from 'react';
import { render } from 'react-testing-library';
import Home from '../Home';

it('Home renders correctly', () => {
  const tree = render(<Home />);
  expect(tree).toMatchSnapshot();
});
