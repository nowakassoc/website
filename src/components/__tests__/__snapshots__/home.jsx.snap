// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        id="section-home"
      >
        <div
          class="home-col-left"
        >
          <span
            class="section-link"
            id="home"
          />
          <h1>
            Nowak and Associates, Inc.
          </h1>
          <h2
            class="tagline"
          >
            Healthcare and Business Consulting
          </h2>
          <p>
            <img
              alt="Martin C. Nowak - Nowak and Associates"
              class="home-portrait-mobile"
              src="test-file-stub"
            />
            Nowak and Associates offers boutique health strategies for MDs and hospitals. Our services include personalized advice for best doctors and hospitals, legal work for medical malpractice and medical class action lawsuits, business plan development and inventor guidance.
            <br />
            <a
              href="#contact"
              id="home-contact"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-envelope fa-w-16 icon-link"
                data-icon="envelope"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
                  fill="currentColor"
                />
              </svg>
              Contact Nowak & Associates
            </a>
          </p>
        </div>
        <div
          class="home-col-right"
        >
          <img
            alt="Martin C. Nowak - Nowak and Associates"
            class="home-portrait"
            src="test-file-stub"
          />
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      id="section-home"
    >
      <div
        class="home-col-left"
      >
        <span
          class="section-link"
          id="home"
        />
        <h1>
          Nowak and Associates, Inc.
        </h1>
        <h2
          class="tagline"
        >
          Healthcare and Business Consulting
        </h2>
        <p>
          <img
            alt="Martin C. Nowak - Nowak and Associates"
            class="home-portrait-mobile"
            src="test-file-stub"
          />
          Nowak and Associates offers boutique health strategies for MDs and hospitals. Our services include personalized advice for best doctors and hospitals, legal work for medical malpractice and medical class action lawsuits, business plan development and inventor guidance.
          <br />
          <a
            href="#contact"
            id="home-contact"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-envelope fa-w-16 icon-link"
              data-icon="envelope"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
                fill="currentColor"
              />
            </svg>
            Contact Nowak & Associates
          </a>
        </p>
      </div>
      <div
        class="home-col-right"
      >
        <img
          alt="Martin C. Nowak - Nowak and Associates"
          class="home-portrait"
          src="test-file-stub"
        />
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
